tests/ui/type-alias-impl-trait/escaping-bound-var.stderr

```diff
@@ -10,6 +10,14 @@ note: lifetime declared here
 LL | pub type Foo = impl for<'a> Trait<'a, Assoc = impl Test<'a>>;
    |                         ^^
 
-error: aborting due to 1 previous error
+error: unconstrained opaque type
```

Comment:
And this one too, do you know why it occurs?

Reply:
I'll look into it.